command/agent/internal/ctmanager/runner_config.go

```diff
@@ -0,0 +1,149 @@
+package ctmanager
```

Comment:
Should we move the package to `ctrunner` or just `runner` instead?

Reply:
eh, lets leave for now. if we ever get away from consul-template maybe it would make more sense